src/layout/Home.js

```diff
@@ -2,11 +2,15 @@ import React, { useEffect } from 'react';
 import { useStore } from 'store/index';
 import ItemsContainer from 'components/item/ItemsContainer';
 
-const Home = () => {
+const Home = ({ sexFitFilter = 'clearFilter' }) => {
     const { state, dispatch } = useStore();
 
     useEffect(() => {
-        dispatch({ type: 'clearFilter' })
+      dispatch({ type: sexFitFilter });
```

Comment:
when the route changes filter the products based on the route name, could also use `withRouter` here I guess.